addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -68,6 +68,9 @@
             </t>
         </t>
     </xpath>
+    <xpath expr="//button[hasclass('o_wslides_js_channel_unsubscribe')]" position="attributes">
+        <attribute name="t-att-data-enroll">channel.enroll</attribute>
```

Comment:
Actually you could directly add enroll in website_slides as the field exists in the base module. That way we have less inheritance.